great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
+	position: absolute;
+	top: 45px;
+	right: 175px;
+	
+}
+
+.services-header {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: row;
+	margin: auto;
+	
+}
+
+.services-main {
+	display: flex;
+	flex-wrap: nowrap;
+	font-size: 32px;
+	justify-content: flex-start;
+	padding-left: 315px;
+	padding-top: 24px;
+}
+
+.services-text {
+	display: flex;
+	flex-wrap: nowrap;
+	font-size: 18px;
+	justify-content: flex-start;
+	padding-left: 315px;
+	padding-top: 10px;
+}
+
+.digital-design {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding-left: 315px;
+	padding-top: 24px;
+}
+
+.digital-design #border-main {
+	border: 1px solid black;
+	background-color: #F1EDED;
+	height: 200px;
+	width: 440px;
+	padding-left: 12px;
+}
+
+.digital-design h2 {
+	font-size: 16px;
+	font-weight: bold;
+	padding-left: 10px;
+	padding-top: 10px;
+}
+
+.digital-design .main-p {
+	font-size: 14px;
+	font-style: normal;
+	 
+}
+
+.digital-design p {
+	padding-top: 16px;
+	padding-left: 16px;
+	font-weight: bold;
+	font-size: 12px; 
+}
+
+.digital-design #border-button {
+	border: 1px thin black;
+	background-color: #FCF7F7;
+	height: 40px;
+	width: 100px;
+}
+
+
+/* --------------- */
+
+
+
+
+.digital-marketing {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding-left: 315px;
+	padding-top: 24px;
+}
+
+.digital-marketing #border-main {
+	border: 1px solid black;
+	background-color: #F1EDED;
+	height: 200px;
+	width: 440px;
+	padding-left: 12px;
+}
+
+.digital-marketing h2 {
+	font-size: 16px;
+	font-weight: bold;
+	padding-left: 10px;
+	padding-top: 10px;
+}
+
+.digital-marketing .main-p {
+	font-size: 14px;
+	font-style: normal;
+	 
+}
+
+.digital-marketing p {
+	padding-top: 16px;
+	padding-left: 16px;
+	font-weight: bold;
+	font-size: 12px; 
+}
+
+.digital-marketing #border-button {
+	border: 1px thin black;
+	background-color: #FCF7F7;
+	height: 40px;
+	width: 100px;
+}
+
+
+
+/* --------------- */
+
+.ios-development {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding-left: 315px;
+	padding-top: 24px;
+}
+
+.ios-development #border-main {
+	border: 1px solid black;
+	background-color: #F1EDED;
+	height: 200px;
+	width: 440px;
+	padding-left: 12px;
+}
+
+.ios-development h2 {
+	font-size: 16px;
+	font-weight: bold;
+	padding-left: 10px;
+	padding-top: 10px;
+}
+
+.ios-development .main-p {
+	font-size: 14px;
+	font-style: normal;
+	 
+}
+
+.ios-development p {
+	padding-top: 16px;
+	padding-left: 16px;
+	font-weight: bold;
+	font-size: 12px; 
+}
+
+.ios-development #border-button {
+	border: 1px thin black;
+	background-color: #FCF7F7;
+	height: 40px;
+	width: 100px;
+}
+
+
+.facts {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	flex-direction: column;
+	padding-left: 315px;
+	padding-top: 24px;
+}
+
+
+ul {
+	list-style-type: square;
+	margin: 10px;
+}
+
+.services-info {
```

Comment:
Try to use flexbox for this selector and the footer, absolute positioning here makes the style unnecessarily brittle.